Formula/peg-markdown.rb

```diff
@@ -3,8 +3,7 @@ class PegMarkdown < Formula
   homepage "https://github.com/jgm/peg-markdown"
   url "https://github.com/jgm/peg-markdown/archive/0.4.14.tar.gz"
   sha256 "111bc56058cfed11890af11bec7419e2f7ccec6b399bf05f8c55dae0a1712980"
-  # license ["GPL-2.0", "MIT"] - pending https://github.com/Homebrew/brew/pull/7953
-  license "MIT"
+  license any_of: ["GPL-2.0", "MIT"]
```

Comment:
```suggestion
  license any_of: ["GPL-2.0-or-later", "MIT"]
```

From the [`LICENSE` file](https://github.com/jgm/peg-markdown/blob/0e0e9e5d7b5e133bd28849e1d72e9636a3b49109/LICENSE#L10-L13):

> This program is free software; you can redistribute it and/or modify
> it under the terms of the GNU General Public License as published by
> the Free Software Foundation; either version 2 of the License, or
> (at your option) any later version.